pages/blog/governance-motivation.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Finding a Good Open Governance Model for AsyncAPI"
+date: 2021-03-08T06:00:00+01:00
+type: Strategy
+tags:
+  - governance
+  - foundation
+  - openness
+cover: /img/posts/governance-motivation-cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+  - name: Fran Mendez
+    photo: /img/avatars/fmvilas.webp
+    link: https://twitter.com/fmvilas
+    byline: AsyncAPI Founder
+excerpt: 'AsyncAPI can be successful if the initiative is community-driven. To be community-driven, we need the community to see it can drive things and make an impact. For that, we need a proper open governance model. What model would be the best?'
+featured: true
+---
+
+> tl;dr charter for the AsyncAPI Initiative open governance is ready for the review. Leave your comments there in [this](https://github.com/asyncapi/.github/pull/37) pull request. We will wait two weeks for your feedback.
+
+December 2020 brought to AsyncAPI important announcement, a [partnership with Postman](/blog/asyncapi-partners-with-postman). This huge milestone secured the AsyncAPI Initiative development efforts as few active community members moved to Postman. This move speeded up our work on transferring AsyncAPI to a foundation and forming an open governance model to assure the community that a single company does not control AsyncAPI Initiative.
+
+It took us a lot of time to write down the initial charter for the initiative.
+
+> **Pro Tip**:
+[Charter](https://dictionary.cambridge.org/dictionary/english/charter) is not a person that creates charts :smiley:
+Nobody's trying to be a smart ass here. We actually had to google that :smiley:
+
+## The vision
+
+We started with a basic vision in our heads and studied governance models of many diverse communities, like NodeJS Foundation, OpenJS Foundation, GraphQL Foundation, Cloud Events, OpenAPI Initiative, and CNCF. We also got a charter template from a foundation. That was a lot to handle. 
+
+<img className="w-1/3" src="https://media.giphy.com/media/5QMPpCzH6yxEqhev5A/giphy.gif" />
```

Comment:
I dont know.. A small image to the left looks off in some way, not sure you see the same? 😄

Reply:
I centered it now, any better? or are you saying that the image is unnecessary in general?

Reply:
Don't get me wrong I love the image 😄 The centering is much better 👍